sections/featured-collection.liquid

```diff
@@ -30,28 +30,32 @@
   endif
 
   assign columns_mobile_int = section.settings.columns_mobile | plus: 0
-  assign show_mobile_slider = false
-  if section.settings.swipe_on_mobile and products_to_display > columns_mobile_int
-    assign show_mobile_slider = true
+  assign show_slider_on_mobile = false
+  if section.settings.enable_mobile_slider and products_to_display > columns_mobile_int
+    assign show_slider_on_mobile = true
+  endif
```

Comment:
it was mostly to reduced the amount of times it was written and make it easier to read and know what the conditions are for. Either show the slider on desktop or/and mobile. 
I don't think it'd be hard to replace tho 🤷 
Curious to hear what @sofiamatulis thinks of it during her review 🤔

Reply:
I've been thinking more about this. If it was used once or twice, I'd keep my suggestion to remove this assign as there are only two conditions being checked.

However, because one condition (`show_desktop_slider`) appears 6 times on the file and the other one appears 11 times (`show_mobile_slider`), I think the assigns end up helping with readability.

Please ignore my original suggestion and keep them! :D